theme.js

```diff
@@ -0,0 +1,42 @@
+//Add transition class DOM and remove after 500ms
+let trans = () => {
+    document.documentElement.classList.add('transition');
+        window.setTimeout(() => {
+            document.documentElement.classList.remove('transition')
+        }, 500)
+}
+
+var isClicked;
```

Comment:
Might be good to name this in a way that would indicate that it overrides the system theme.